simulators/gaia/__init__.py

```diff
@@ -151,104 +157,110 @@ def _execute(self, msg):
-        return '%d%c' % (x, self.tail)
+        return x
 
     def _setg(self, args):
         x = args[0]
         y = args[1]
         self.VG[x - 1] = y
-        return '%d%c' % (x, self.tail)
+        return x
 
-    def _setsg(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _setsg(args):
+        return args[0]
 
-    def _setsd(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _setsd(args):
+        return args[0]
 
-    def _setsgz(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _setsgz(args):
+        return args[0]
 
-    def _setsdz(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _setsdz(args):
+        return args[0]
 
-    def _savecpu(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _savecpu(args):
+        return args[0]
 
-    def _resetd(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _resetd(args):
+        return args[0]
 
-    def _resetg(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _resetg(args):
+        return args[0]
 
-    def _save(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _save(args):
+        return args[0]
 
-    def _setdf(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _setdf(args):
+        return args[0]
 
-    def _setgf(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _setgf(args):
+        return args[0]
 
-    def _getef(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _getef(args):
+        return args[0]
 
-    def _enable(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _enable(args):
+        return args[0]
 
-    def _disable(self, args):
-        x = args[0]
-        return '%d%c' % (x, self.tail)
+    @staticmethod
+    def _disable(args):
+        return args[0]
 
     def _getvg(self, args):
         x = args[0]
-        return '%d%c' % (self.VG[x - 1], self.tail)
+        return self.VG[x - 1]
 
     def _getvd(self, args):
         x = args[0]
-        return '%d%c' % (self.VD[x - 1], self.tail)
+        return self.VD[x - 1]
 
-    def _getid(self, _):
-        # corrente tra 0 e XmA
-        return '0%c' % self.tail
+    @staticmethod
+    def _getid(_):
+        return 0  # current in mA, inside range [0, X]
 
-    def _getref(self, args):
+    @staticmethod
+    def _getref(args):
         x = args[0]
         if x == 1:
-            return '2.5' + self.tail
-        return '5' + self.tail
+            return '2.5'
+        return 5
```

Comment:
One return value is a string, the other one is an integer. Probably they both get formatted correctly in the answer but I think they should both be integers just like other methods/functions